### PR TITLE
Iss hipcms 89 write content

### PR DESCRIPTION
**Pull request by maltehol**

### CODE REVIEW

File: app/topics/content/content.component.ts [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+    </div>
+  </div>
+`
+})
+export class ContentComponent implements OnInit {
+  private currentUser: User;
+  private topicId: number;
+
+  apiUrl = '';
+  public config = {
```
Comment (janis-kra) [inline app/topics/content/content.component.ts]: not sure if it is necessary to mock the whole config object, but isn't much of an issue

File: app/topics/content/content.component.ts [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+    let urlDiff_arr = this.config.setHistoryData.urlDiff;
+
+    this.docEditor.setHistoryData({
+      version: version,
+      url: url_arr[version - 1] != "" ? url_arr[version - 1] : null,
+      urlDiff: urlDiff_arr[version - 1] != "" ? urlDiff_arr[version - 1] : null,
+    });
+  }
+
+  private getOnlyOffice() {
```
Comment (janis-kra) [inline app/topics/content/content.component.ts]: as this function does not reall return anything, it shouldnt be named as if it was a getter (better: 'loadOnlyOffice')

File: app/topics/content/content.component.ts [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+      onRequestHistory: () => this.onRequestHistory(),
+      onRequestChangeHistoryData: (data: any) => this.onRequestChangeHistoryData(data)
+    };
+  }
+
+  ngOnInit() {
+    this.topicId = this.route.snapshot.params['id'];
+    this.ooApiService.getUrl('/topic/' + this.topicId + '/exists', {}).toPromise()
+      .then(
+        (response: any) => { this.getOnlyOffice(); }
```
Comment (janis-kra) [inline app/topics/content/content.component.ts]: curly braces may be omitted when only one expression is called

File: app/topics/content/content.component.ts (hunk 1 above)
Comment (maltehol) [inline app/topics/content/content.component.ts]: It helps to avoid ts lint errors and auto-completion.

**Pull request merged**